### PR TITLE
Issue2 subscriptions interface

### PR DESCRIPTION
**Pull request by DeeZone**
- http request to `mb-user-api` to get current subscriptions settings
- check for `banned` account
- Display user `subscriptions1 setting as checkboxes in form
- check for missing `targetEmail` value as URL parameter.

**To Test**:
- VPN connection to DS network
- start node app: `npm start`
- With browser: `http://localhoast:3000?targetEmail=<email address>`
  - The "email address" will have `subscriptions` settings that the form will render

### CODE REVIEW

File: lib/userDetails.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+ */
+var http = require('http'),
+    util = require('util');
+
+module.exports = {
+
+  getDetails: function(req, callbackRes) {
+
+    var options = {
+      hostname: 'localhost',
```
Comment (deadlybutter) [inline lib/userDetails.js]: wouldnt this need to change for it to work on non dev environment?

File: lib/userDetails.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+      var data = '';
+      response.on('data', function(chunk) {
+        data += chunk;
+      });
+      response.on('error', function() {
+        console.log('Error making http GET request to ' + options.hostname + ':' + options.port + options.path);
+      });
+      response.on('end', function() {
+        if (data.length) {
```
Comment (deadlybutter) [inline lib/userDetails.js]: hmm, did you test this? I dont think this would work -- just tried it in a node terminal and it threw an undefined error

Comment (DeeZone) [inline lib/userDetails.js]: You'll need either `mb-user-api` running with a connection to the mongo `mb-users` db. I should have detailed this in the PR.

File: views/home.handlebars [hunk 3]
```diff
@@ -1,23 +1,32 @@
...
+          <input type="checkbox" id="user_events" name="user_events" value="user_events" {{subscriptions_user_events}} >
+          <span class="option__indicator"></span>
+          <span>User Events - Birthday and Annual DoSomething.org registration anniversaries.</span>
+        </label>
+        <label class="option -checkbox">
+          <input type="checkbox" id="digest" name="digest" value="digest" {{subscriptions_digest}} >
+          <span class="option__indicator"></span>
+          <span>Weekly Digest of campaigns you've signed up for.</span>
+        </label>
+        <button class="button" type=submit">Update</button>
```
Comment (deadlybutter) [inline views/home.handlebars]: Missing a ", Should be type="submit"

File: lib/userDetails.js (hunk 1 above)
Comment (DeeZone) [inline lib/userDetails.js]: Good point, I'll update this with the live IP. The catch is you'll need a VPN connection for it to work. Details in the PR description on the way.

**Pull request merged**